src/mono/wasm/build/WasmApp.Native.targets

```diff
@@ -181,6 +183,7 @@
       <_EmccCFlags Include="-DLINK_ICALLS=1"                   Condition="'$(WasmLinkIcalls)' == 'true'" />
       <_EmccCFlags Include="-DCORE_BINDINGS" />
       <_EmccCFlags Include="-DGEN_PINVOKE=1" />
+      <_EmccCFlags Include="-emit-llvm" />
 
```

Comment:
Won't this be passed to the .bc -> .o builds as well ?

Reply:
So, looks like I broke this in the last few days! Adding here is fine, because it will get used only to compile the `.c` files. And the bitcode files will use the LDFlags.

I've added a crude test to check for this, so we don't regress again.